### PR TITLE
Changed my caption

### PR DESCRIPTION
**Pull request by andrew-scoppa**
This PR closes #9

### CODE REVIEW

**Comment by vwang2020** [comment]
Looks good.

File: _slides/02-andrew-scoppa.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Dino Cat!
```
Comment (pandeyvishal007) [inline _slides/02-andrew-scoppa.md:6]: Looks good to me..but what's Dino??

Comment (vwang2020) [inline _slides/02-andrew-scoppa.md:6]: Is Dino a Cat? I thought it's a dog :)

File: _slides/02-andrew-scoppa.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
-![dinotocat](https://octodex.github.com/images/dinotocat.png)
+![dinotocat](https://octodex.github.com/images/dinotocat.png)
```
Comment (pandeyvishal007) [inline _slides/02-andrew-scoppa.md:8]: You forgot to change this

File: _slides/02-andrew-scoppa.md (hunk 1 above)
Comment (vwang2020) [reply _slides/02-andrew-scoppa.md:6]: Verified. It's  Cat.

Comment (andrew-scoppa) [reply _slides/02-andrew-scoppa.md:6]: This is from the flinstones cartoon

**Review by pandeyvishal007 — APPROVED** [review]
Reviewed and Done 😄

**Pull request merged**